src/components/Forecast.tsx

```diff
@@ -9,27 +10,68 @@ interface Props {
 	units?: Unit
 }
 
-export const Forecast: React.FC<Props> = ({forecast, units = Unit.Standard}) => {
-	const tempUnit = (): string => {
-		switch (units) {
-			case Unit.Metric: return 'ºC';//'&#8451;'
-			case Unit.Imperial: return 'ºF';//'&#8457;'
-			default: return 'K';//'&#8490;'
+export const Forecast: React.FC<Props> = ({forecast, units}) => {
+	
+	const [newUnits,setNewUnits]=useState<string>('ºC');
+	
+	const [newUnit,setNewUnit]=useState(Math.floor(forecast.main.temp_max)/Math.floor(forecast.main.temp_min));	
+	
+	const [celsius, setCelsius] = useState(units===Unit.Metric?true:false);
+	const [kelvin, setKelvin] = useState(units===Unit.Standard?true:false);
+	const [fahrenheit, setFahrenheit] = useState(units===Unit.Imperial?true:false);
+
+	const convertTempUnit = (event: React.ChangeEvent<HTMLInputElement>) => {
+		let newUnit:string=event.target.name;		
+		let ans:number=Math.floor(forecast.main.temp_max)/Math.floor(forecast.main.temp_min);
+		switch (newUnit) {
+		      case Unit.Metric:  
+		      	 setCelsius(true);
+				 setFahrenheit(false);
+				 setKelvin(false);
+				 setNewUnits("ºC");
+				 break;
+		      case Unit.Imperial: 
+		      	 ans=(ans * 9/5)+32;
+				 setFahrenheit(true);
+				 setKelvin(false);
+				 setCelsius(false);
+				 setNewUnits("ºF");
+				 break;
+		      default: 
+		      	 ans=ans + 273.15;
+				 setKelvin(true);
+				  setFahrenheit(false);
+				 setCelsius(false);
+				 setNewUnits("K");
+				 break;
 		}
+		setNewUnit(ans);
 	}
+	
 
 	return (
 		<div className="current-forecast">
 			<ConditionImage weather={forecast.weather[0]} />
 			<div className="location-name">
-				{forecast.name}
+				<h3>
+					{forecast.name}
+				</h3>
 			</div>
 			<div className="details">
 				<span className="condition">{forecast.weather[0].main} </span>
 				<span className="temperatures">
-					{Math.floor(forecast.main.temp_max)}/{Math.floor(forecast.main.temp_min)}{tempUnit()}
+					{newUnit}{newUnits}
```

Comment:
Here i want to show the max temperature and the minimum. Also, it has to be rounded using `Math.floor` because it's showing to many numbers.
![image](https://user-images.githubusercontent.com/59806216/136588050-84ee6d29-8bca-44a6-8dd8-afe33a6a9c83.png)